Include/internal/pycore_code.h

```diff
@@ -443,6 +444,12 @@ write_obj(uint16_t *p, PyObject *val)
     memcpy(p, &val, sizeof(val));
 }
 
+static inline void
+write_void(uint16_t *p, void *val)
```

Comment:
This is very similar to `write_obj` above.
Maybe have one function, `write_ptr(uint16_t *p, void *val)`, and use that in place of `write_obj` and `write_void`?

Reply:
Wouldn't that require casting?

Reply:
`void *` takes any pointer